gluoncv/model_zoo/model_store.py

```diff
@@ -128,6 +128,7 @@
     ('6a25eeceb7d27bd9c05fa2bf250c55d3960ad4c7', 'resnet50_v1d_8.8x'),
     ('a872796b63fb883116831db3454711421a628154', 'resnet101_v1d_1.9x'),
     ('712fccb185921a596baebe9246ff6c994b88591b', 'resnet101_v1d_2.2x'),
+    ('529096cb961cc4bbe450553329633443cf05dd25', 'alpha_pose_resnet101_v1b_coco'),
```

Comment:
@Jeff-sjtu I added the weights converted previously, can you verify if it's the one you are using as well?

Reply:
I check the sha1 with `shasum xxx.params`

Reply:
Thanks for your comment. I have updated the code.